kolibri/plugins/video_mp4_render/assets/src/vue/index.vue

```diff
@@ -74,14 +78,17 @@
       },
 
       setPlayState(state) {
+        this.recordProgress();
         if (state === true) {
           this.videoPlayer.$('.videotoggle').classList.add('videopaused');
           this.videoPlayer.$('.videoreplay').classList.add('display');
           this.videoPlayer.$('.videoforward').classList.add('display');
+          this.$emit('startTracking', this.Kolibri);
         } else {
           this.videoPlayer.$('.videotoggle').classList.remove('videopaused');
           this.videoPlayer.$('.videoreplay').classList.remove('display');
           this.videoPlayer.$('.videoforward').classList.remove('display');
+          this.$emit('stopTracking', this.Kolibri);
```

Comment:
Does `this.Kolibri` need to be passed here, or can the parent (listening) component add it into the action itself, from its own reference?